docs/Reference/CheckBox.md

```diff
@@ -825,6 +825,7 @@ export default class Plugin extends React.Component {
     }
 
     render() {
+        return
```

Comment:
Syntax error:
return
         \<div> .....\<\/div>
Need change to:
return (
         \<div>  .....\<\/div>
);

Reply:
@hokimcuc0202 
That's my fault. I forget the '{}' for return statement. I had added it. 

Thank you a lot.